### PR TITLE
BZ#1616161 - added a note that firewalld is not supported on Atomic Host

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://bugzilla.redhat.com/show_bug.cgi?id=1616161

### CODE REVIEW

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -1316,11 +1316,24 @@ the Ansible host file at install:
...
 [OSEv3:vars]
-os_firewall_use_firewalld=True
+os_firewall_use_firewalld=True <1>
 ----
+<1> Setting this variable to `true` opens the required ports and adds rules to
+the default zone, ensuring that firewalld is configured correctly.
+
+[IMPORTANT]
+====
+*firewalld* is not supported on Atomic Host. But you can use it by supplying an additional `openshift_enable_unsupported_configurations` parameter.
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc]: s/Host. But/Host, but
s/parameter./parameter in the Ansible host file:  (or other correct file)

**Review by kalexand-rh — COMMENTED** [review]
Couple of picks, but LGTM.

File: install/configuring_inventory_file.adoc (hunk 1 above)
Comment (sdodson) [reply install/configuring_inventory_file.adoc]: I don't think we should mention the unsupported options here.

**Pull request merged**